target/linux/bcm5862x/files/arch/arm/boot/dts/bcm958625-mx64a0.dts

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/*
+ * Device Tree Bindings for Cisco Meraki MX64(A0).
+ *
+ * Copyright (C) 2021 Matthew Hagan <mnhagan88@gmail.com>
+ */
+
+/dts-v1/;
+
+#include "bcm958625-mx64x.dtsi"
+
+/ {
+	model = "Cisco Meraki MX64(A0)";
+	compatible = "meraki,mx64", "brcm,bcm58625", "brcm,nsp";
```

Comment:
meraki,mx64a0 ?

Reply:
Yes this is very necessary to prevent users upgrading from an mx64a0 to mx64 and vice versa, which may make the device inaccessible and require reinstall by usb.